src/components/templates/FaqTemplate.js

```diff
@@ -1,10 +1,54 @@
-import React from 'react';
-import { Link } from 'gatsby';
+import React from "react";
+import { Link } from "gatsby";
+import Heading from "../atoms/Heading";
+import Fontain from "../atoms/Fontain";
+import FontainImg from "../../../static/assets/Fontain.png";
+import FaqPair from "../molecules/FaqPair";
+import styled from "@emotion/styled";
+
+const ContentContainer = styled.div({
```

Comment:
rename it to `ContentWrapper` or `FaqWrapper`.. it will be used specifically on this page.. also, add mobile and atablet styles